### PR TITLE
🏗 Very significantly speed up travis builds

### PR DESCRIPTION
**Pull request by cramforce**
By turning off caching 🤷🏼‍♀️

## Before
Screen Shot 2020-06-11 at 10 23 46

## After
Screen Shot 2020-06-11 at 10 23 33

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -126,20 +126,3 @@ jobs:
     - script: unbuffer node build-system/pr-check/performance-tests.js # See #28148
     - script: unbuffer node build-system/pr-check/remote-tests.js # See #28343
   fast_finish: true
-before_cache:
-  # do not store cache for pr builds or experiment stage builds
-  - if [[ $TRAVIS_EVENT_TYPE == pull_request ]] || [[ $TRAVIS_BUILD_STAGE_NAME == experiment ]]; then exit $TRAVIS_TEST_RESULT ; fi
-cache:
```
Comment (rsimha) [inline .travis.yml:132]: With this, we can also delete all the `CACHE_NAME` environment settings across this file. (They were used to separate caches from different jobs.)

Comment (cramforce) [reply .travis.yml:132]: Done

File: .travis.yml [hunk 2]
```diff
@@ -99,47 +94,23 @@ jobs:
...
-    - build-system/tasks/e2e/node_modules
-    - build-system/tasks/performance/node_modules
-    - build-system/tasks/visual-diff/node_modules
-    - sauce_connect
-    - validator/node_modules
-    - validator/nodejs/node_modules
-    - validator/webui/node_modules
-    - validator/java/bazel-installer
-    - $HOME/.m2
-    - .karma-cache
```
Comment (rsimha) [inline .travis.yml:144]: This dir was being cached in order to speed up transforms during tests, which will now have to be done from scratch for every build.

/cc @jridgewell

Comment (cramforce) [reply .travis.yml:144]: Maybe we could use the mechanism that shares stuff between build stages to build our own cache.

Comment (rsimha) [reply .travis.yml:144]: Good idea. Definitely possible.

**Pull request merged**